libraries/provider_dot_my_cnf.rb

```diff
@@ -0,0 +1,35 @@
+class Chef
+  class Provider
+    # used to create and managed .my.cnf files
+    class MysqlmDotMyCnf < Chef::Provider::LWRPBase
+      use_inline_resources if defined?(use_inline_resources)
+
+      def whyrun_supported?
+        true
+      end
+
+      def action_create
```

Comment:
We could also just document that we ship a template, and let people use the existing/built-in `template` resource, which comes with those actions. This feels like such a thin wrapper and it isn't adding much.